website/content/docs/internals/high-availability.mdx

```diff
@@ -47,7 +47,41 @@ the request, the request is forwarded to the active server. Read-only requests a
 Like traditional HA standbys, a Performance Standby Node becomes the active instance when the active node is sealed, fails, or loses
 network connectivity.
 
-# Tutorial 
+# Becoming an active node
+
+The active node in an HA configuration is the node that holds an HA lock. When a node relinquishes active duty, either
+because it's unavailable or because an operator has invoked [`vault operator step-down`](/vault/docs/commands/operator/step-down),
+all the nodes in the cluster will attempt to grab the HA lock. Only one node will succeed and this node will become the new active
+node.
```

Comment:
```suggestion
Active nodes in an HA configuration holds an HA lock. When active nodes become
unavailable or step down because an operator invokes 
[`vault operator step-down`](/vault/docs/commands/operator/step-down), all the 
nodes in the cluster attempt to grab the HA lock. The first node that succeeds 
in grabbing and holding the lock becomes the new active node.
```

Style correction: write in active voice